core/Network/TLS/Extension.hs

```diff
@@ -268,8 +268,8 @@ instance Extension SessionTicket where
     extensionEncode (SessionTicket {}) = runPut $ return ()
     extensionDecode _ = runGetMaybe (return SessionTicket)
 
-data HeartBeat = HeartBeat HeartBeatMode
-    deriving (Show,Eq)
+newtype HeartBeat = HeartBeat HeartBeatMode
+                  deriving (Show,Eq)
```

Comment:
I'm not a fan of aligning the deriving clause this way.

Reply:
OK. Other `newtype`s use one line for each. I will change in the same way.